lib/model/sign_up.rb

```diff
@@ -0,0 +1,23 @@
+class SignUp < Sequel::Model(:userdetails)
+  dataset_module do
+    def all
```

Comment:
not sure .all describes exactly what is going on here.
.last_two_days would be more descriptive IMO

Reply:
That's everything earlier than tomorrow – a way to ensure we don't use corrupted data from the future.

So, yeah, it ain't `all`, you're right. But it's not `.last_two_days` either.

Reply:
`*everything earlier than tomorrow* == `.current` ?

Reply:
Or not... `current` sounds like *logged in right now* 👎